<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admin</class>
 <widget class="QWidget" name="admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>950</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>12</y>
     <width>166</width>
     <height>576</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>600</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <widget class="QPushButton" name="info_button">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>60</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: #FFFFFF;
	font: 12pt &quot;黑体&quot;;
  	text-align:center;
	border:1px outset rgb(255, 255, 255);
	border-bottom:1px solid #e0e0e0;
}

QPushButton:hover {
	background-color: rgb(229, 229, 229);
	border-left:5px solid rgb(150, 150, 150);
	border-bottom:1px solid #e0e0e0;
	padding-left:5px;
}

QPushButton:pressed {
	border-left:5px solid rgb(130, 130, 130);
	border-bottom:1px solid #e0e0e0;
	padding-left:5px;
}</string>
    </property>
    <property name="text">
     <string>管理员中心</string>
    </property>
   </widget>
   <widget class="QPushButton" name="renew_button">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>210</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>60</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: #FFFFFF;
	font: 12pt &quot;黑体&quot;;
  	text-align:center;
	border:1px outset rgb(255, 255, 255);
	border-bottom:1px solid #e0e0e0;
}

QPushButton:hover {
	background-color: rgb(229, 229, 229);
	border-left:5px solid rgb(150, 150, 150);
	border-bottom:1px solid #e0e0e0;
	padding-left:5px;
}

QPushButton:pressed {
	border-left:5px solid rgb(130, 130, 130);
	border-bottom:1px solid #e0e0e0;
	padding-left:5px;
}</string>
    </property>
    <property name="text">
     <string>电影上下架</string>
    </property>
   </widget>
   <widget class="QPushButton" name="assign_button">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>270</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>60</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: #FFFFFF;
	font: 12pt &quot;黑体&quot;;
  	text-align:center;
	border:1px outset rgb(255, 255, 255);
	border-bottom:1px solid #e0e0e0;
}

QPushButton:hover {
	background-color: rgb(229, 229, 229);
	border-left:5px solid rgb(150, 150, 150);
	border-bottom:1px solid #e0e0e0;
	padding-left:5px;
}

QPushButton:pressed {
	border-left:5px solid rgb(130, 130, 130);
	border-bottom:1px solid #e0e0e0;
	padding-left:5px;
}</string>
    </property>
    <property name="text">
     <string>影院排片</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exit_button">
    <property name="geometry">
     <rect>
      <x>42</x>
      <y>390</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>27</y>
      <width>191</width>
      <height>111</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>150</height>
     </size>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;方正舒体&quot;;
border-bottom:2px solid #e0e0e0;
image: url(:/new/prefix1/image/MOVIEHANDY.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>188</x>
     <y>12</y>
     <width>750</width>
     <height>576</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>750</width>
     <height>600</height>
    </size>
   </property>
   <widget class="QWidget" name="page"/>
   <widget class="QWidget" name="page_2"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
